Newspack/Newspack/Folders/FolderManager.swift

```diff
@@ -165,4 +164,69 @@ class FolderManager {
 
         return folders
     }
+
+    /// Move the folder at the specified URL to a new location.
+    ///
+    /// - Parameters:
+    ///   - source: The location of the folder to move.
+    ///   - destination: The new location for the folder.
+    /// - Returns: true if successful, false otherwise
+    ///
+    func moveFolder(at source: URL, to destination: URL) -> Bool {
+        let name = sanitizeFolderName(name: destination.lastPathComponent)
+        guard isValidFolderName(name: name) else {
+            return false
+        }
+
+        let destination = destination.deletingLastPathComponent().appendingPathComponent(name, isDirectory: true)
+        guard folderExists(url: source) && !folderExists(url: destination) else {
+            return false
+        }
+
+        do {
+            try fileManager.moveItem(at: source, to: destination)
+            return true
+        } catch {
+            LogError(message: "Error moving folder \(source) to \(destination)")
+        }
+
+        return false
+    }
+
+    /// Rename the folder at the specified url to the specified name.
+    ///
+    /// - Parameters:
+    ///   - source: The url of the folder to rename.
+    ///   - name: The new name.
+    /// - Returns: The new URL of the folder, or nil if the folder could not be
+    /// renamed.
```

Comment:
Annoying comment of the month: Maybe `renamed` belongs to L201? (since there are already lines longer than this one?)

Reply:
Yeah.  I struggle with returning at the 80 character mark or just letting it roll over and flip flop on which I prefer.  I'll leave it for now but might revisit in another PR. :) 
Thanks Jorge!